requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
hmm, that complicates matters. Well at any rate I think it would be a bad idea for us to depend on a git version of Theano, for now. When people do `pip install Theano`, 0.7 is the version they will get.